packages/engine-handlebars/lib/engine_handlebars.js

```diff
@@ -129,8 +129,8 @@ const engine_handlebars = {
    * assume it's already present
    */
   spawnMeta: function (config) {
-    this.spawnFile(config, '_head.hbs');
-    this.spawnFile(config, '_foot.hbs');
+    this.spawnFile(config, '_head.' + config.patternExtension);
```

Comment:
This could be dangerous.
When pattern lab discovers multiple pattern engines, it will try to spawn the file found in the engine's folder located under `_meta/head...`
In that case, the file will not be found, and an error will be thrown here https://github.com/pattern-lab/patternlab-node/blob/4cf86b610721b55dec8390d08771528dbc20952d/packages/engine-handlebars/lib/engine_handlebars.js#L116
You can either catch the error and ignore it, or you revert it to its former state.

Another issue could be, that by chance the file will be taken, but the output for the twig file will be overwritten by the content of the `hbs` file. In that case, it could lead to an error when serving pattern lab because it has a different rendering engine.

Reply:
I would rather look into the `engineFileExtensions` and when the `config.patternExtension` is included there, output the required file. (In that case, we need to read the file we deliver with the engine from the _meta folder and spawn it as the configured extension in `config.patternExtension`.